README.md

```diff
@@ -24,6 +24,25 @@ import 'details-dialog-element'
 </details>
 ```
 
+## Deferred loading
+
+Dialog content can be loaded from a server by embedding an [`<include-fragment>`][fragment] element.
+
+[fragment]: https://github.com/github/include-fragment-element/
```

Comment:
Unless we're going to keep these link references in the bottom, we could just as well inline the link href.